### PR TITLE
Crosspost SSI's Blog Post

### PR DESCRIPTION
**Pull request by NPDebs**
In this pull request, a new post has been added to the website in response to a request by @pherterich.
The content, originally shared by the [Software Sustainability Institute](https://www.software.ac.uk/), is authored by R. Camacho Toro, Y. Briceño, and A. Martínez. An image related to the post has also been included, as in the original post.

reina-blog

*Note for reviewer:*
- [x] Please, confirm proper formatting and inclusion of relevant metadata.
- [x] Confirm that the included image is licensed under CC BY.

Thank you!

### CODE REVIEW

**Comment by NPDebs** [comment]
**Preview [here](https://deploy-preview-728--ols-bebatut.netlify.app/posts)...**

File: _posts/2023-12-07-Latin-American-students-drive-innovation-with-open-data.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+authors:
+- camachoreina
+image: https://www.software.ac.uk/sites/default/files/2023-12/unnamed.jpg
+photos:
+  name: Image by Reina Camacho Toro
+  license: CC-BY
+  url: https://github.com/camachoreina
+---
+
+_This blog post was original shared by Software Sustainability Institute, and found [here](https://www.software.ac.uk/blog/hackathon-co-afina-2023-latin-american-students-drive-innovation-open-data), as authored by R. Camacho Toro, Y. Briceño and A. Martínez._
```
Comment (pherterich) [inline _posts/2023-12-07-Latin-American-students-drive-innovation-with-open-data.md:13]: Slight rewording suggestion, feel free to ignore:
This blog post was original shared by Software Sustainability Institute, and can be found [here](https://www.software.ac.uk/blog/hackathon-co-afina-2023-latin-american-students-drive-innovation-open-data), as authored by R. Camacho Toro, Y. Briceño and A. Martínez.

**Comment by pherterich** [comment]
Tagging @camachoreina directly to let her confirm licence for the image.

File: _posts/2023-12-07-Latin-American-students-drive-innovation-with-open-data.md (hunk 1 above)
Comment (camachoreina) [reply _posts/2023-12-07-Latin-American-students-drive-innovation-with-open-data.md:13]: Hello! Using CC BY is perfect. Shall we added to the figure directly? Or the OLS web already has CC by default?

Comment (camachoreina) [reply _posts/2023-12-07-Latin-American-students-drive-innovation-with-open-data.md:13]: Thanks a lot!

Comment (pherterich) [reply _posts/2023-12-07-Latin-American-students-drive-innovation-with-open-data.md:13]: Thanks @camachoreina - OLS website content is CC-BY-SA unless specified differently, so this is perfect!

**Comment by NPDebs** [comment]
@bebatut **Non-urgent:** All parties approve of the state of this PR. Please merge if you agree. 🙏

File: _posts/2023-12-07-Latin-American-students-drive-innovation-with-open-data.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: Hackathon Co-Afina 2023 - Latin American Students Drive Innovation with Open Data
+authors:
+- camachoreina
```
Comment (bebatut) [inline _posts/2023-12-07-Latin-American-students-drive-innovation-with-open-data.md:5]: I think Alexander Martinez is mxrtinez on the website

Comment (NPDebs) [reply _posts/2023-12-07-Latin-American-students-drive-innovation-with-open-data.md:5]: Found and added him. Thanks!

**Comment by bebatut** [comment]
Thanks @NPDebs and @pherterich

**Pull request merged**